guide/ops/osgi-configuration.md

```diff
@@ -1,79 +1,62 @@
 ---
-title: OSGi Distribution
+title: OSGi Configuration
 layout: website-normal
 ---
 
-# Running Apache Brooklyn inside Karaf container
+Configuration of Brooklyn when running under Karaf is largely done through standard Karaf mechanisms. 
```

Comment:
You should refer to at least the first instance of Brooklyn within a document as Apache Brooklyn